web/viewer.css

```diff
@@ -1238,29 +1356,31 @@ html[dir='rtl'] .attachmentsItem > button {
   color: rgba(255, 255, 255, 0.5);
 }
 .outlineItemToggler::before {
-  content: url(images/treeitem-expanded.png);
+  mask: url(images/treeitem-expanded.svg) no-repeat;
+  -webkit-mask: url(images/treeitem-expanded.svg) no-repeat;
```

Comment:
Given that we (unfortunately) still need to support IE 11 in the default viewer, using `mask` is sadly not going to work (applies throughout the patch); note https://developer.mozilla.org/en-US/docs/Web/CSS/mask#Browser_compatibility